tests/rptest/redpanda_cloud_tests/iceberg_catalogs_test.py

```diff
@@ -0,0 +1,150 @@
+    """
+    Verify that cluster infra/config matches config profile used to launch - only applies to cloudv2
+    """
+    def __init__(self, test_context):
+        super().__init__(test_context=test_context)
+        self._ctx = test_context
+        self._ipClient = InstallPackClient(
+            self.redpanda._cloud_cluster.config.install_pack_url_template,
+            self.redpanda._cloud_cluster.config.install_pack_auth_type,
+            self.redpanda._cloud_cluster.config.install_pack_auth)
+
+    def setUp(self):
+        super().setUp()
+        cloud_cluster = self.redpanda._cloud_cluster
+        self.logger.debug(f"Cloud Cluster Info: {vars(cloud_cluster)}")
+        install_pack_version = cloud_cluster.get_install_pack_version()
+        self._ip = self._ipClient.getInstallPack(install_pack_version)
+        self._clusterId = cloud_cluster.cluster_id
+        self._configProfile = self._ip['config_profiles'][
+            cloud_cluster.config.config_profile_name]
+
+    def test_healthy(self):
+        r = self.redpanda.cluster_unhealthy_reason()
+        assert r is None, r
+        assert self.redpanda.cluster_healthy()
+        self.redpanda.assert_cluster_is_reusable()
+
+    @cluster(num_nodes=1)
+    def test_databricks_basic(self):
+        dbx_ctx = DatabricksContext.from_context(self._ctx)
+
+        cloud_cluster = self.redpanda._cloud_cluster
+        client: RpCloudApiClient = cloud_cluster.public_api
+
+        databricks_client = DatabricksWorkspace(context=self._ctx)
+        bucket = f"redpanda-cloud-storage-{self._clusterId}"
+        catalog_info = databricks_client.create_catalog(bucket=bucket)
+        properties = {"iceberg_enabled": True}
+        enable_resp = cloud_cluster.update_cluster_property_public(
+            self._clusterId, properties)
+        self.logger.debug(f"Enable iceberg response: {enable_resp}")
+
+        # Parameters for creating Redpanda secret
+        secret_id = "UNITY_CLIENT_SECRET"
+        
+        #secret_data = dbx_ctx.client_secret
+        # Access credentials (client_secret, client_id) from dbx_ctx.credentials
+        if isinstance(dbx_ctx.credentials, OauthCredentials):
+            secret_data = dbx_ctx.credentials.client_secret
+            databricks_client_id = dbx_ctx.credentials.client_id
+        else:
+            secret_data = dbx_ctx.credentials.token
+            databricks_client_id = None
+
+        create_resp = cloud_cluster.create_secret(secret_id, secret_data)
+        self.logger.debug(f"Create secret response: {create_resp}")
+
+        iceberg_rest_catalog_endpoint = dbx_ctx.iceberg_rest_url
+        iceberg_rest_catalog_warehouse = dbx_ctx.sql_warehouse_path
+
+        # Construct the payload for the request
+        properties = {
+            "iceberg_rest_catalog_endpoint": iceberg_rest_catalog_endpoint,
+            "iceberg_rest_catalog_authentication_mode": "oauth2" if isinstance(dbx_ctx.credentials, OauthCredentials) else "pat",
```

Comment:
i'm not sure if this is supported at all in cloud but pat is not a valid value, bearer is